### PR TITLE
tests list-addressing bail

### PR DESCRIPTION
**Pull request by joemfb**
forwardports #704

### CODE REVIEW

File: tests/bug/gh-703.hoon [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+::
+/+  *test
+|%
+::
+++  test-hang
+  %-  expect-fail
+    |.  &33:~[[%leaf p="syntax error"] [%leaf p="\{1 11}"]]
+::
+++  test-vere-bail
+  ~&  [`path`% running-unsafe+"bails out with out of loom error"]
```
Comment (eglaysher) [inline tests/bug/gh-703.hoon]:
```
`path`%
```

this is rather ugly; i assume this is to put the path into the stack trace for when vere itself bails, but if vere bails, the sigpam is unlikely to trigger, no?

**Comment by ohAitch** [comment]
Do we still expect hangs/bails here, if the relevant c issue got fixed?

File: tests/bug/gh-703.hoon (hunk 1 above)
Comment (joemfb) [reply tests/bug/gh-703.hoon]: That is ugly -- I meant to remove it. Already removed from the "hang" test case.

**Comment by ohAitch** [comment]
Yep, I would just rename them to "expect-no-hang","expect-no-bail" or sth :P

**Comment by cgyarvin** [comment]
It should just fail

**Pull request merged**